src/react-components/home-root.js

```diff
@@ -275,7 +275,55 @@ class HomeRoot extends Component {
                   </div>
                 </div>
               )}
+              <IfFeature name="show_newsletter_signup">
+                <div id="mc_embed_signup" className={styles.footerContent}>
+                  <form
+                    action="https://mozilla.us19.list-manage.com/subscribe/post?u=a97038e0879516f7b8300d67f&amp;id=ebf17d04e4"
+                    method="post"
+                    id="mc-embedded-subscribe-form"
+                    name="mc-embedded-subscribe-form"
+                    className="validate"
```

Comment:
I don't think we have a `validate` class. Same for the `clear` classNames below.